src/components/Scheme/ContributionRewardExtRewarders/Competition/List.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react";
+import { BreadcrumbsItem } from "react-breadcrumbs-dynamic";
+import { ISchemeState, IDAOState, IProposalState } from "@daostack/client";
+import { SortService } from "lib/sortService";
+import { CompetitionStatusEnum, CompetitionStatus } from "./utils";
+import Card from "./Card";
+import * as css from "./Competitions.scss";
+
+interface IExternalProps {
+  daoState: IDAOState;
+  scheme: ISchemeState;
+  proposals: Array<IProposalState>;
+}
+
+interface IStateProps {
+  statusMap: Map<string, CompetitionStatus>;
+}
+
+type IProps = IExternalProps;
+
+export default class CompetitionsList extends React.Component<IProps, IStateProps> {
+
+  constructor(props: IProps) {
+    super(props);
+    this.state = { statusMap: new Map() };
+  }
+
+  private handleStatusChange = (proposalState: IProposalState, status: CompetitionStatus) => {
+    /**
+     * the Cards maintain, through countdowns, the Copetition status, and provide it here
+     * where we save the status and force a rerender
+     */
+    this.state.statusMap.set(proposalState.id, status);
+    // force render
+    this.setState({ statusMap: this.state.statusMap });
+  }
+
+  private compareCompetitions = (a: IProposalState, b: IProposalState): number => {
+
+    const statusA = this.state.statusMap.get(a.id);
+    const statusB = this.state.statusMap.get(b.id);
+
+    if (!(statusA && statusB)) {
+      return 0;
+    }
+
+    const statusIndexA = Object.values(CompetitionStatusEnum).indexOf(statusA.status);
+    const statusIndexB = Object.values(CompetitionStatusEnum).indexOf(statusB.status);
+    /**
+     * sort by the ordinal position of the CompetitionStatusEnum values
+     */
+    const retval = SortService.evaluateNumber(statusIndexA, statusIndexB);
+    if (retval)
```

Comment:
dont we need to use braces here? has this been linted?

Reply:
it is linted.  apparently breaks no rules.  I added the braces anyway.